src/Castle.Windsor/Castle.Windsor-VS2017.csproj

```diff
@@ -35,4 +35,10 @@
   <PropertyGroup Condition="'$(TargetFramework)'=='net45'">
     <DefineConstants>$(DefineConstants);FEATURE_PERFCOUNTERS;FEATURE_GAC</DefineConstants>
   </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)'=='netstandard1.5'">
```

Comment:
Where does this target framework come from?  Above, this project only declares:

`<TargetFramework>net45</TargetFramework>`